docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,392 @@
+# Getting Started
```

Comment:
Global: technical doc should be written from the first-person perspective (i.e. avoid using `we` and `us`, but rather use `you`).